osbenchmark/builder/utils/artifact_variables_provider.py

```diff
@@ -0,0 +1,24 @@
+ARCH_MAPPINGS = {
+    "x86_64": "x64",
+    "aarch64": "arm64"
+}
+
```

Comment:
Similar to how you made an Enum to replace usage of hardcoded "opensearch", is there a reason why ARCH_MAPPINGS can't be done similarly?

Reply:
No reason it can't be, probably better than a file-level constant in case it gets reused elsewhere. Will refactor